Mid.css

```diff
@@ -0,0 +1,214 @@
+body
+{ 
+      background-color: white;
+}
+.resume
+{     
+    margin:0 auto;
+
```

Comment:
why this extra space and indentation is not proper

Reply:
Done.